spec/redis_spec.rb

```diff
@@ -403,6 +449,21 @@
     expect(@namespaced.hgetall("foo")).to eq({"key1" => "value1"})
   end
 
+  it "should utilize connection_pool while adding namespace to pipelined blocks" do
```

Comment:
how do we know connection pool is being used in this test? Can we assert something on the connection pool instance?

Reply:
Added assertions on ConnectionPool calls 👍